tests/nested/manual/refresh-from-old-base/task.yaml

```diff
@@ -0,0 +1,72 @@
+summary: Test refreshing from old snapd and base prior to required symlink
+
+details: |
+    Base snaps used to not initialize /snap/snapd/current symlink to seed.
+    However the question is what happens when starting with an old image,
+    and updating snapd before the base. This makes sure that this initial
+    symlink is not involved with updates.
+
+systems:
+  - ubuntu-22.04-64
+
+environment:
+  NESTED_BUILD_SNAPD_FROM_CURRENT: false
+  NESTED_REPACK_BASE_SNAP: false
+  NESTED_FORCE_SNAKEOIL_KEYS: true
+
+prepare: |
+    snap download --channel="latest/${BASE_CHANNEL}" core22 --basename=core22-new
+
+    # shellcheck source=tests/lib/prepare.sh
+    . "$TESTSLIB"/prepare.sh
+    rm -rf new-snap/
+    mkdir new-snap/
+    build_snapd_snap new-snap/
+    for f in new-snap/snapd_*.snap; do
+        mv "${f}" snapd-new.snap
+    done
+    rm -rf new-snap
+
+    # 1033 does not contain the work around for missing
+    # /snap/snapd/current symlink during seeding. See
+    # https://github.com/snapcore/core-base/pull/185
+    snap download --revision=1033 core22 --basename=core22-old
+    # We need to repack to avoid auto refresh
```

Comment:
How does this avoid auto referesh? Is it because we have some new timestamps in the mix?

Can you perhaps touch a dummy new file to ensure that even if unpack / repack produce reproducible binary the introduction of the new file make the content different.

Reply:
I have no idea. We have always done like that in the spread tests. There is multiple things like that in `tests/lib`. I suspect that if a snap is not asserted by the store, then it does not have a store version. And it does not get updated. So it is probably just the hash has to be different.

Reply:
And yes, I suppose it has to do with time when creating the squashfs.

Reply:
https://github.com/canonical/snapd/blob/2df91e6dfec90653ba3a1464b76ef3bcf8201c21/tests/lib/prepare.sh#L1318

Reply:
I have added a file.